src/components/Section/Relationships/People/People.jsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react'
+import { i18n } from '../../../../config'
+import { Accordion, ValidationElement, Svg } from '../../../Form'
+import Person from './Person'
+import { PeopleValidator } from '../../../../validators'
+import SummaryProgress from '../../History/SummaryProgress'
+import PeopleCounter from './PeopleCounter'
+import { dateRangeFormat } from './../../Psychological/summaryHelper'
+
+export default class People extends ValidationElement {
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      List: props.List,
+      errorCodes: []
+    }
+
+    this.update = this.update.bind(this)
```

Comment:
@bmallred explaining the` bind` method might be a good topic to cover in a tech chat.